lint-pr-title/action.yml

```
@@ -0,0 +1,39 @@
+name: Lint PR Title
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)